data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -586,12 +586,16 @@ def _workforce_factor(self) -> bool:
         )
 
         # Now, calculate workforce criteria for island territories.
+        island_areas_workforce_eligibility_columns = [
+            field_names.ISLAND_AREAS_UNEMPLOYMENT_LOW_HS_EDUCATION_FIELD,
+            field_names.ISLAND_AREAS_POVERTY_LOW_HS_EDUCATION_FIELD,
+            field_names.ISLAND_AREAS_LOW_MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD,
+        ]
 
-        # F a couple of values, create a combined field and criteria field.
         # First, combine unemployment.
         (
             self.df,
-            unemployment_island_areas_criteria_field_name,
```

Comment:
just renaming these to put `island_areas` first like our other field names.

Reply:
good call